jest/gen-config.js

```diff
@@ -3,14 +3,34 @@
  */
 var fs = require('fs');
 var path = require('path');
-var testPaths = ['src', 'plugins', 'tests', 'packages'];
+var roots = ['src', 'tests'];
 
+/**
+ * Adding all packages and plugins as separate root directories 
```

Comment:
Nit: I think this comment should be cleaned up/clarified, it's long and a mentions a few things. I suggest moving each part closer to the code that is directly related!